src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RemoteProjectSnapshotProjectEngineFactory.cs

```diff
@@ -69,6 +69,7 @@ public void Configure(RazorCodeGenerationOptionsBuilder options)
             // We don't need to explicitly subscribe to options changing because this method will be run on every parse.
             options.IndentSize = _optionsMonitor.CurrentValue.TabSize;
             options.IndentWithTabs = !_optionsMonitor.CurrentValue.InsertSpaces;
+            options.RemapDesignTimeLinePragmaPathsOnWindows = true;
```

Comment:
Is this change visible in tests? Or why is it needed?

Reply:
This is how we turn it on for the LSP editor. I'm not aware of any tests that directly cover it. @davidwengier Any ideas?

Reply:
No idea, I'm afraid. Probably not covered